controllers/cronset_controller.go

```diff
@@ -68,14 +71,50 @@ func (r *CronSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{}, err
 	}
 
-	// for testing
-	r.Log.Info(obj.GetObjectKind().GroupVersionKind().Group)
-	r.Log.Info(obj.GetObjectKind().GroupVersionKind().Version)
-	r.Log.Info(obj.GetObjectKind().GroupVersionKind().Kind)
+	var nodeSelector map[string]string
```

Comment:
If it's possible, it's better to divide each changes by one PR. It would be better to review one by one. :)

Reply:
I agree. Hmm.. Should I split the PR again now, or can we do an offline review?

Reply:
If you don't mind, I think It's pretty better to split PR. we should make a PR for only one modification.

Reply:
Okay, I will do that